src/set_theory/cardinal.lean

```diff
@@ -1044,6 +1044,35 @@ begin
     exact not_lt.mp (mt (mul_lt_omega_iff_of_ne_zero hx1 hy1).mp (λ h, not_lt.mpr hx2 h.1)) },
 end
 
+lemma nonempty_to_nat_eq_zero_infinite {α : Type u} [nonempty α] :  (#α).to_nat = 0 → ω ≤ (#α) :=
```

Comment:
These lemmas are no longer used in this PR, so you might consider splitting them off into a separate PR.

Reply:
Done